demo/hostvolume/example-plugin-mkfs

```diff
@@ -6,6 +6,11 @@
 
 set -euo pipefail
 
+version='0.0.1'
+fingerprint() {
+  echo '{"version": "'"$version"'"}'
```

Comment:
I'd consider using `printf` here as you've done for the other outputs below, just to avoid this ugly escaping.

Reply:
oh yeah! I think you did that `printf`, actually. I always forget that it's useful for this kinda thing.